### ISSUE TITLE
Calendar view does not update when deleteing the last event view

### ISSUE THREAD

**Issue by opw0011** [richardtop/CalendarKit#48]
I have implemented a feature to delete a event by long pressing a event view.
Basically, the deletion works perfectly when there are more than 1 event in the data array. 
However, when I delete the last event, the calendar view does not update immediately and the deleted view just stay there. I need to swipe to next week and come back to that day. Then, deleted event will then be disappeared.

I tried to reload the view after the deletion using the following functions, but none of it works.
```swift
self.reloadInputViews()
self.reloadData()
self.dayView.reloadData()
self.dayView.reloadInputViews()
````

I am not sure if it is a bug. Do you have a example code on how to delete a event properly?

**Comment by richardtop** [comment]
@opw0011, thanks for reporting! Seems that it's a bug. I will be able to check this bug during this week. However, if you have a fix or proposal, feel free to submit a pull request.
Also, I'd like to see the example code on how you're deleting events, or an simple test-project which has minimum amount of code to completely reproduce the issue.

**Comment by whitefc87** [comment]
In my opinion, this problem occur when event data is empty.
how to solve this bug?

**Comment by richardtop** [comment]
Fixed in #63
